### PR TITLE
Initial docker infrastructure

### LINKED ISSUE #1: Create initial Dockerfile and docker-compose.yml
... and document in the readme getting up and running with local development.

Note: The existing DAGs and harvester code in this PR is for example only and is expected to be removed as work proceeds.

### PR DESCRIPTION
**Pull request by aaron-collier**
Fixes #1 

This:

- [x] Creates the initial docker-compose.yml and Dockerfile
- [x] Adds initial (example) DAGs to establish structure
- [x] Add initial README with startup instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+docker build . -f Dockerfile --tag suldlss/dlme-airflow:latest
+```
+
+then stop and restart all of the docker compose resources.
+
+## Visit the Airflow dashboard
+
+open your browser to `http://localhost:8080`
+
+## Enable the DAGs you wish to run locallay
```
Comment (jermnelson) [inline README.md:34]: typo: should be `locally`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,36 @@
...
+
+## Run inidividual DAGs
```
Comment (jermnelson) [inline README.md:36]: typo: should be `individual`

File: dlme_airflow/dags/aims_harvester.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+# The DAG object; we'll need this to instantiate a DAG
+from airflow import DAG
+
+# Operators; we need this to operate!
+from airflow.operators.bash import BashOperator
+from airflow.operators.python import PythonOperator
+from airflow.utils.dates import days_ago
+
+import sys
```
Comment (jermnelson) [inline dlme_airflow/dags/aims_harvester.py:12]: I think in a follow-up PR we could either do one of the follow:

- In the Dockerfile add `/opt/dlme_airflow/` to the PYTHONPATH variable
- Allow `dlme_airflow` to be installed as a python package using a binary wheel distribution with pip, see  [documentation](https://airflow.apache.org/docs/apache-airflow/stable/modules_management.html#creating-a-package-in-python)

File: README.md (hunk 1 above)
Comment (aaron-collier) [reply README.md]: Fixed. Thanks for the catch, my spelling isn't going to get any better ;-)

File: dlme_airflow/dags/aims_harvester.py (hunk 3 above)
Comment (aaron-collier) [reply dlme_airflow/dags/aims_harvester.py]: Done, thanks!

**Review by jermnelson — APPROVED** [review]

**Pull request merged**